lib/file-info-view.coffee

```diff
@@ -46,7 +46,7 @@ class FileInfoView
     text = @getActiveItemCopyText(copyRelativePath)
     @copiedTooltip = atom.tooltips.add @element,
       title: "Copied: #{text}"
-      trigger: 'click'
+      trigger: 'manual'
```

Comment:
What was the purpose of this change? According to the docs, it looks like this makes the tooltip appear as soon as it's created, with no way to hide it until the tooltip itself is disposed?

Reply:
When I moved the event listener to `@element`, the `Copied: ...` tooltip no longer displayed . I am not certain, but I think this is because previously:
1. A single click event handled on `@currentPath` added the tooltip.
2. The same click event then bubbled to the parent `@element`, triggering the tooltip's display.

Whereas now, the tooltip is added by `@element` while handling the click, after the opportunity to trigger the tooltip with that same click.

As for the documented behavior of `manual`, we were using a 0 delay before this anyway, so the tooltip was already showing immediately, and the same click handler that adds the tooltip sets a timeout to dispose of it after 2 seconds. This is my first time using the tooltip manager though, so if `manual` is problematic for some reason I'm all ears.